<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterForm</class>
 <widget class="QWidget" name="RegisterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>511</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>511</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>511</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mi Deporte Movil</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/img/ico2.ico</normaloff>:/img/ico2.ico</iconset>
  </property>
  <widget class="QLineEdit" name="leEmail">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>190</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#leEmail{
	padding: 1px;
	border-style: solid;
	border: 2px solid gray;
	border-radius: 8px;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePassword">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>190</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lePassword{
	padding: 1px;
	border-style: solid;
	border: 2px solid gray;
	border-radius: 8px;
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePhone">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>190</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lePhone{
	padding: 1px;
	border-style: solid;
	border: 2px solid gray;
	border-radius: 8px;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leFirstName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>190</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#leFirstName{
	padding: 1px;
	border-style: solid;
	border: 2px solid gray;
	border-radius: 8px;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leLastName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>190</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#leLastName{
	padding: 1px;
	border-style: solid;
	border: 2px solid gray;
	border-radius: 8px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Myungjo Std M</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Myungjo Std M</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Myungjo Std M</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Nombre</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Myungjo Std M</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Apellido</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Myungjo Std M</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Telefono</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbRegister">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>110</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Myungjo Std M</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#pbRegister {
	color: #333;
	border: 2px solid #555;
	border-radius: 11px;
	padding: 5px;
	background: qradialgradient(cx: 0.3, cy: -0.4,
	fx: 0.3, fy: -0.4,
	radius: 1.35, stop: 0 #fff, stop: 1 #888);
	min-width: 80px;
}
#pbRegister:hover {
	background: qradialgradient(cx: 0.3, cy: -0.4,
	fx: 0.3, fy: -0.4,
	radius: 1.35, stop: 0 #fff, stop: 1 #bbb);
}
#pbRegister:pressed {
	background: qradialgradient(cx: 0.4, cy: -0.1,
	fx: 0.4, fy: -0.1,
	radius: 1.35, stop: 0 #fff, stop: 1 #ddd);
}</string>
   </property>
   <property name="text">
    <string>Registrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Castellar</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>430</y>
     <width>110</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Myungjo Std M</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#btnCancelar {
	color: #333;
	border: 2px solid #555;
	border-radius: 11px;
	padding: 5px;
	background: qradialgradient(cx: 0.3, cy: -0.4,
	fx: 0.3, fy: -0.4,
	radius: 1.35, stop: 0 #fff, stop: 1 #888);
	min-width: 80px;
}
#btnCancelar:hover {
	background: qradialgradient(cx: 0.3, cy: -0.4,
	fx: 0.3, fy: -0.4,
	radius: 1.35, stop: 0 #fff, stop: 1 #bbb);
}
#btnCancelar:pressed {
	background: qradialgradient(cx: 0.4, cy: -0.1,
	fx: 0.4, fy: -0.1,
	radius: 1.35, stop: 0 #fff, stop: 1 #ddd);
}</string>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>511</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="img.qrc">:/img/register1.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_7</zorder>
  <zorder>leEmail</zorder>
  <zorder>lePassword</zorder>
  <zorder>lePhone</zorder>
  <zorder>leFirstName</zorder>
  <zorder>leLastName</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>pbRegister</zorder>
  <zorder>label_6</zorder>
  <zorder>btnCancelar</zorder>
 </widget>
 <tabstops>
  <tabstop>leEmail</tabstop>
  <tabstop>lePassword</tabstop>
  <tabstop>leFirstName</tabstop>
  <tabstop>leLastName</tabstop>
  <tabstop>lePhone</tabstop>
  <tabstop>pbRegister</tabstop>
 </tabstops>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
